Create sigfox-agent-4-0-0-improvements-in-endpoint-authorization-and-permissions.md (#4487)
SoftwareAG/c8y-docs: content/change-logs/device-management/sigfox-agent-4-0-0-improvements-in-endpoint-authorization-and-permissions.md

---
date: ""
title: Improvements in endpoint authorization and permissions
product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-TCiiCOknp5
    label: LPWAN
build_artifact:
  - value: tc-CB45dexyZ
    label: sigfox-agent
ticket: DM-5344
version: 4.0.0
---

> Description:  The required user permissions to read, create, or update Sigfox connections under Administration > Connectivity have been updated. To read the Sigfox connections, users must have at least the Sigfox READ permission, and to create/update/delete connections, users require the Sigfox ADMIN permission. In the Device Management application, this affects the LPWAN configuration tab in the device details, as the Current connection field requires READ permission. You can assign the necessary permissions to users in Administration > Accounts > User, Roles.  Moreover, the sigfox 'callback' endpoints such as 'sigfoxDataCallback', 'sigfoxErrorCallback' etc. require the Sigfox ADMIN permission or at least the Inventory READ permission.

Reviewer: ```suggestion
To improve security and access control for Sigfox device connectivity, the permission requirements for managing Sigfox connections have been clarified and enforced.

Users now need specific Sigfox permissions to work with connections under **Administration > Connectivity**. To view Sigfox connections, users require the Sigfox READ permission, while creating, updating, or deleting connections requires the Sigfox ADMIN permission. Additionally, Sigfox callback endpoints such as `sigfoxDataCallback` and `sigfoxErrorCallback` now require either the Sigfox ADMIN permission or the Inventory READ permission.

This change affects users accessing the **LPWAN configuration** tab in device details within the Device Management application, as viewing the **Current connection** field now requires READ permission. Administrators can assign the necessary permissions to users in **Administration > Accounts > User, Roles**.

For details, see [Sigfox](/device-integration/lpwan/#sigfox).
```

---

> date: ""

Reviewer 2: ```suggestion
date: '2026-04-02'
```

as it's already deployed to eu-latest.

> title: Improvements in endpoint authorization and permissions

Reviewer 2: Shouldn't we always add the Service name to the title as well for better visibility?

```suggestion
title: Sigfox service improvements in endpoint authorization and permissions
```
